tests/console/kdump_and_crash.pm

```diff
@@ -33,20 +33,32 @@ sub run {
     activate_kdump;
 
     # restart to activate kdump
-    power_action('reboot');
+    check_var('BACKEND', 'spvm') ? power_action('reboot', keepconsole => 1) : power_action('reboot');
+    reconnect_mgmt_console if check_var('BACKEND', 'spvm');
     $self->wait_boot;
     select_console 'root-console';
 
+    # WIP: FIX IT
+    systemctl 'status kdump';
+    script_run 'echo $?';
```

Comment:
why ?

Reply:
WIP section to fix

Reply:
Fixed by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7457